build-system/tasks/storybook-react/index.js

```diff
@@ -0,0 +1,36 @@
+('use strict');
+
+const argv = require('minimist')(process.argv.slice(2));
+const {build: buildBinary} = require('../build');
+const {buildEnv, launchEnv} = require('../storybook');
+const {createCtrlcHandler} = require('../../common/ctrlcHandler');
+const {defaultTask: runAmpDevBuildServer} = require('../default-task');
+/**
+ * @return {Promise<void>}
+ */
+async function storybookReact() {
+  // React dist needs to be comparied before running
+  await buildBinary();
+  const {build = false, 'storybook_env': env = 'amp,preact'} = argv;
+  const envs = env.split(',');
+  if (!build && envs.includes('amp')) {
+    await runAmpDevBuildServer();
+  }
+  if (!build) {
+    createCtrlcHandler('storybook');
```

Comment:
I think this needs to be the name of the command:
```suggestion
    createCtrlcHandler('storybook-react');
```